Utils/Accessibility.swift

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+
+public protocol AccessibilityActionsSource: class {
+    func accessibilityCustomActionsForView(view: UIView) -> [UIAccessibilityCustomAction]?
+}
+
+public class Action {
```

Comment:
I know it is a bit unfortunate to put it in a file called `Accessibility.swift` then. But I could not think of a more suitable name :-)